test/format/ops.c

```diff
@@ -530,9 +530,16 @@ ops(void *arg)
 				ckpt_config = ckpt_name;
 			}
 
-			testutil_checkfmt(
-			    session->checkpoint(session, ckpt_config),
-			    "%s", ckpt_config == NULL ? "" : ckpt_config);
+			ret = session->checkpoint(session, ckpt_config);
+			/*
+			 * We may be trying to create a named checkpoint while
+			 * we hold a cursor open to the previous checkpoint.
+			 * In that case we'll get back EBUSY.  Tolerate EBUSY
+			 * if we have the readonly cursor open.
+			 */
+			if (ret != 0 && (ret != EBUSY || !readonly))
+				testutil_die(ret, "%s",
+				    ckpt_config == NULL ? "" : ckpt_config);
```

Comment:
@sueloverso, I'd suggest tolerating EBUSY in general, is there any reason to fine-grain that conditional?